docs/install/kubernetes.md

```diff
@@ -145,7 +145,7 @@ locally in order to log in and manage templates.
    helm install coder coder-v2/coder \
        --namespace coder \
        --values values.yaml \
-       --version 2.9.4
+       --version 2.10.2
```

Comment:
@stirby another place to automate with the release script.

Reply:
good catch